.yamato/_templates.yml

```diff
@@ -43,383 +47,459 @@ test_hdrp_template_Win_trunk:
+        image: package-ci/mac:stable
+        flavor: m1.mac
+    commands:
+      -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
+      -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
+      -  .Editor/Unity.app/Contents/MacOS/Unity -projectPath com.unity.template-hd -batchmode -quit
+    artifacts:
+        primed:
+            paths:
+              -  "com.unity.template-hd/Library/Artifacts/**"
+              -  "com.unity.template-hd/Library/ArtifactDB"
+              -  "com.unity.template-hd/Library/SourceAssetDB"
+pack_hdrp_template_OSX_trunk:
+    name: Pack HDRP Template OSX trunk
     agent:
         type: Unity::VM
         image: package-ci/win10:stable
         flavor: b1.large
+    commands:
+      -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+      -  upm-ci template pack --project-path com.unity.template-hd
     dependencies:
-      -  .yamato/_templates.yml#pack_hdrp_template
-      -  .yamato/_packages.yml#pack_core
-      -  .yamato/_packages.yml#pack_shadergraph
-      -  .yamato/_packages.yml#pack_vfx
-      -  .yamato/_packages.yml#pack_config
-      -  .yamato/_packages.yml#pack_hdrp
-      -  .yamato/_editor_priming.yml#editor:priming:trunk:Win
+      -  .yamato/_templates.yml#pre_pack_hdrp_template_OSX_trunk
+    artifacts:
+        packages:
+            paths:
+              -  "upm-ci~/packages/**/*"
+              -  "upm-ci~/templates/**/*"
+test_hdrp_template_OSX_trunk:
+    name: Test HDRP Template OSX trunk
+    agent:
+        type: Unity::VM::osx
+        image: package-ci/mac:stable
+        flavor: m1.mac
+    dependencies:
+      -  .yamato/_templates.yml#pack_hdrp_template_OSX_trunk
+      -  .yamato/_editor_priming.yml#editor:priming:trunk:OSX
     commands:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
       -  unity-downloader-cli --source-file unity_revision.txt -c editor --wait --published-only
-      -  for /r packages_temp %%x in (*.tgz) do copy %%x upm-ci~\packages
-      -  copy upm-ci~\packages\*.tgz .Editor\Data\Resources\PackageManager\Editor
-      -  upm-ci template test -u .\.Editor --type updated-dependencies-tests --project-path com.unity.template-hd
+      -  cp ./upm-ci~/packages/*.tgz ./.Editor/Unity.app/Contents/Resources/PackageManager/Editor
+      -  upm-ci template test -u $(pwd)/.Editor/Unity.app/Contents/MacOS/Unity --project-path com.unity.template-hd
     artifacts:
         logs:
             paths:
               -  "**/test-results/**"
-test_universal_template_Win_trunk:
-    name: Test Universal Template Win trunk
+publish_hdrp_template_trunk:
```

Comment:
is it worth adding some dry-run publish jobs?

Reply:
That would be ideal, but from the docs the `publish` command does not handle the --dry-run argument:
```
upm-ci template publish [-h]
  [--project-path PROJECT_PATH]
  [--artifacts-path ARTIFACTS_PATH]
```
That said I realize it is handled in the package context. So maybe it's just an oversight and it is available in the template context as well. Let me try

Reply:
It works, I guess that was just missing in the docs :) Thanks for catching that